.circleci/deploy-config.yml

```diff
@@ -2,9 +2,9 @@
 version: 2.1
 
 orbs:
-  node: circleci/node@5.1.0
-  slack: circleci/slack@4.12.5
-  aws-cli: circleci/aws-cli@4.0.0
+  node: circleci/node@5.2.0
+  slack: circleci/slack@4.13.3
+  aws-cli: circleci/aws-cli@4.1.3
```

Comment:
Maybe a silly question, but how did you decide what to bump the orbs to? is it the just the most recent minor[1] version? 

[1] dunno if it's semantic but I mean 'y' in the x.y.z version number.

Reply:
Yeah. I couldn't really find any upgrade advice on breaking changes in the latest majors of the orbs so I decided not to pull that thread here. I'm not sure if doing this has actually done much of use tbh.